cmd/config/config.go

```diff
@@ -366,9 +356,9 @@ func LookupCreds(kv KVS) (auth.Credentials, error) {
 	if err := CheckValidKeys(CredentialsSubSys, kv, DefaultCredentialKVS); err != nil {
 		return auth.Credentials{}, err
 	}
-	accessKey := env.Get(EnvAccessKey, kv.Get(AccessKey))
-	secretKey := env.Get(EnvSecretKey, kv.Get(SecretKey))
-	if accessKey == "" && secretKey == "" {
+	accessKey := kv.Get(AccessKey)
```

Comment:
Will `kv` load it from the environment variable/config file?

Reply:
Yes it does, this code path is only for situations where env is not available